src/explorer/AzureAccountTreeItem.ts

```diff
@@ -68,6 +70,17 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
         await this.trialAppNode?.refresh();
     }
 
+    private async setContext(treeItem: AzExtTreeItem | undefined): Promise<void> {
+        if (!treeItem) {
+            await commands.executeCommand('setContext', 'hasTrialApp', false);
+            return;
+        }
+        await commands.executeCommand('setContext', 'trialAppExpired', treeItem instanceof ExpiredTrialAppTreeItem);
+        if (treeItem instanceof TrialAppTreeItem || treeItem instanceof ExpiredTrialAppTreeItem) {
```

Comment:
Do you need this check?  I assume if there's no treeItem, then the treeItem has to be either the TrialApp or ExpiredTrial tree item types.

Reply:
It could be an invalid tree item if there was an error.